From 37d2115a6bf58c8d34a95cc6407b2807db2b6991 Mon Sep 17 00:00:00 2001
From: Michal Simek <michal.simek@xilinx.com>
Date: Tue, 20 Nov 2012 15:28:49 +0100
Subject: [PATCH] arm: zynq: Remove unused SCU_GLOBAL_TIMER_BASE

It should be the part of this patch
"arm: zynq: Removed scu global timer driver for cleanup"
(sha1: 47fa64c788eccdda66c3625507c36257f5df1bd2)

Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Acked-by: John Linn <john.linn@xilinx.com>
---
 arch/arm/mach-zynq/include/mach/zynq_soc.h |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-zynq/include/mach/zynq_soc.h b/arch/arm/mach-zynq/include/mach/zynq_soc.h
index 4f17df2..25f5af7 100644
--- a/arch/arm/mach-zynq/include/mach/zynq_soc.h
+++ b/arch/arm/mach-zynq/include/mach/zynq_soc.h
@@ -31,7 +31,6 @@
 /* The following are intended for the devices that are mapped early */
 
 #define SCU_PERIPH_BASE			IOMEM(SCU_PERIPH_VIRT)
-#define SCU_GLOBAL_TIMER_BASE		(SCU_PERIPH_BASE + 0x200)
 
 #define SLCR_BASE_VIRT			0xf8000000
 #define SLCR_BASE_PHYS			0xf8000000
-- 
1.7.1

